contrib/charts/cert-manager/values.yaml

```diff
@@ -38,6 +38,10 @@ extraArgs: []
   # supporting resources required for each ClusterIssuer (default is kube-system)
   # - --cluster-resource-namespace=kube-system
 
+extraEnv: []
+# - name: VAULT_CAPATH
+#   value: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
```

Comment:
Could you change this example to not include the VAULT_CAPATH var?

Whilst I understand the reasoning for it right now, the ca path should be set on a per-issuer basis and I would not want to confuse users into thinking this is the best way to set the variable.